tests/testthat/test-build-home-index.R

```diff
@@ -25,3 +25,12 @@ test_that("package repo verification", {
   )
 })
 
+test_that("page title",{
+  pkg <- test_path("assets/pagetitle")
```

Comment:
It seems a bit weird to create a whole directory for this single test. Can you think of a simpler approach?

Reply:
Ok, but not sure the approach I use now is much better?

I thought of using mocking but then when looking at `as_pkgdown()` that I'd have mocked I saw it returned its argument if it were already a pkgdown object, so used that feature.

Reply:
Yes, this is much better. I wouldn't expect a large test when you're making such a small change to the code.

Reply:
Noted for next time! 😉